cli/demo.go

```diff
@@ -0,0 +1,332 @@
+// Copyright 2016 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+// Author: Raphael 'kena' Poss (knz@cockroachlabs.com)
+
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var genDemoCmd = &cobra.Command{
```

Comment:
I think a better description than `demo` would be `example-data`.

Also, I imagine we might want more than one example (something which generates random data, and a lot of it would be great). So I think you'll want to give this command a flag which specifies the flavor of the example data. For now, the default would be `startrek`.